docs/models/openai.md

```diff
@@ -10,7 +10,7 @@ pip/uv-add "pydantic-ai-slim[openai]"
 
 ## Configuration
 
-To use `OpenAIModel` with the OpenAI API, go to [platform.openai.com](https://platform.openai.com/) and follow your nose until you find the place to generate an API key.
+To use `OpenAIChatModel` with the OpenAI API, go to [platform.openai.com](https://platform.openai.com/) and follow your nose until you find the place to generate an API key.
```

Comment:
I think we should explain `OpenAIChatModel` vs `OpenAIResponsesModel` right away, so that people pick the latter unless they're using a Chat Completions-compatible API, or need a feature Responses doesn't support like audio.